frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,107 @@
+  healthOfMCG,
+  resourceProviders,
+  noobaaCount,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: testName,
+    name: 'test-bucket',
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Tests at least one noobaa bucket is present', async () => {
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Creates an Object Bucket Claim and tests equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -n ${testName} -f -`);
+    await browser.sleep(10000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -n ${testName} -f -`);
+  });
+});
+
+describe('Tests Status Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks if Multi Cloud Gateway is in a healthy state', async () => {
+    await browser.wait(until.visibilityOf(healthOfMCG));
+    expect(healthOfMCG.isPresent()).toBe(true);
+  });
+});
+
+describe('Tests Object Data Reduction Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
+  });
+
+  it('Checks for savings value to be in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(savingsValue));
+    const savVal = await savingsValue.getText();
+    const [savDigits, ,] = savVal.split('%');
+    const numSav = Number(savDigits);
+    if (Number.isNaN(numSav)) {
+      expect(savVal.trim()).toEqual('No Savings');
+    } else {
+      expect(numSav).toBeGreaterThanOrEqual(0);
+    }
+  });
+});
+
+describe('Tests Resource Providers card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks for resource provider to have at least 1 AWS resource', async () => {
```

Comment:
What if noobaa deployed on vmware?

Reply:
kubevirt tests just can't be run in our local deployments.